test/gtest/p4runtime.cpp

```diff
@@ -379,7 +381,7 @@ TEST(P4Runtime, P4_16_MatchFields) {
 #if 0
 // Disabling this test because it does not work without installation
 // see issue https://github.com/p4lang/p4c/issues/924
-TEST(P4Runtime, P4_14_MatchFields) {
+TEST_F(P4Runtime, P4_14_MatchFields) {
```

Comment:
How do you feel about fixing this test?

Reply:
I plan to fix that in a later patch in this series. We need to just make GTest use the same code for finding included headers that the normal compiler binary uses.